<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Locations</class>
 <widget class="QMainWindow" name="Locations">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>558</width>
    <height>516</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout_3">
    <item>
     <widget class="QScrollArea" name="scrollArea">
      <property name="widgetResizable">
       <bool>true</bool>
      </property>
      <widget class="QWidget" name="scrollAreaWidgetContents">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>525</width>
         <height>456</height>
        </rect>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_4">
        <item>
         <layout class="QVBoxLayout" name="verticalLayout">
          <item>
           <widget class="QTableWidget" name="Tbw_Locations">
            <property name="selectionMode">
             <enum>QAbstractItemView::SingleSelection</enum>
            </property>
            <property name="selectionBehavior">
             <enum>QAbstractItemView::SelectRows</enum>
            </property>
           </widget>
          </item>
          <item>
           <layout class="QVBoxLayout" name="verticalLayout_2">
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout">
              <item>
               <widget class="QFrame" name="Frm_MainBtns">
                <property name="frameShape">
                 <enum>QFrame::StyledPanel</enum>
                </property>
                <property name="frameShadow">
                 <enum>QFrame::Raised</enum>
                </property>
                <layout class="QHBoxLayout" name="horizontalLayout_5">
                 <item>
                  <widget class="QPushButton" name="Btn_Send">
                   <property name="text">
                    <string>Send</string>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="QPushButton" name="Btn_Distance">
                   <property name="text">
                    <string>Distance</string>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="QPushButton" name="Btn_Reset">
                   <property name="text">
                    <string>&amp;Reset</string>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="QPushButton" name="Btn_Rename">
                   <property name="text">
                    <string>Re&amp;name</string>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="QPushButton" name="Btn_Exit">
                   <property name="text">
                    <string>&amp;Exit</string>
                   </property>
                  </widget>
                 </item>
                </layout>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_3">
              <item>
               <widget class="QFrame" name="Frm_Rename">
                <property name="frameShape">
                 <enum>QFrame::StyledPanel</enum>
                </property>
                <property name="frameShadow">
                 <enum>QFrame::Raised</enum>
                </property>
                <layout class="QHBoxLayout" name="horizontalLayout_4">
                 <item>
                  <widget class="QLineEdit" name="Txt_Name"/>
                 </item>
                 <item>
                  <widget class="QPushButton" name="Btn_Save">
                   <property name="text">
                    <string>&amp;Save</string>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="QPushButton" name="Btn_Cancel">
                   <property name="text">
                    <string>&amp;Cancel</string>
                   </property>
                  </widget>
                 </item>
                </layout>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <layout class="QVBoxLayout" name="verticalLayout_5">
              <item>
               <widget class="QFrame" name="Frm_Distance">
                <property name="frameShape">
                 <enum>QFrame::StyledPanel</enum>
                </property>
                <property name="frameShadow">
                 <enum>QFrame::Raised</enum>
                </property>
                <layout class="QVBoxLayout" name="verticalLayout_6">
                 <item>
                  <layout class="QVBoxLayout" name="verticalLayout_7">
                   <item>
                    <layout class="QHBoxLayout" name="horizontalLayout_6">
                     <item>
                      <layout class="QVBoxLayout" name="verticalLayout_9">
                       <item>
                        <widget class="QLabel" name="label">
                         <property name="text">
                          <string>Latitude :</string>
                         </property>
                        </widget>
                       </item>
                       <item>
                        <widget class="QLabel" name="label_3">
                         <property name="text">
                          <string>Longtitude</string>
                         </property>
                        </widget>
                       </item>
                      </layout>
                     </item>
                     <item>
                      <layout class="QVBoxLayout" name="verticalLayout_10">
                       <item>
                        <widget class="QLabel" name="Lbl_LatitudeS">
                         <property name="text">
                          <string>0.0000</string>
                         </property>
                        </widget>
                       </item>
                       <item>
                        <widget class="QLabel" name="Lbl_LongtitudeS">
                         <property name="text">
                          <string>0.0000</string>
                         </property>
                        </widget>
                       </item>
                      </layout>
                     </item>
                     <item>
                      <layout class="QVBoxLayout" name="verticalLayout_11">
                       <item>
                        <widget class="QPushButton" name="Btn_SelectFromTableS">
                         <property name="text">
                          <string>Selected</string>
                         </property>
                        </widget>
                       </item>
                       <item>
                        <widget class="QPushButton" name="Btn_SelectMyLocationS">
                         <property name="text">
                          <string>Current</string>
                         </property>
                        </widget>
                       </item>
                      </layout>
                     </item>
                    </layout>
                   </item>
                   <item>
                    <layout class="QHBoxLayout" name="horizontalLayout_7">
                     <item>
                      <layout class="QHBoxLayout" name="horizontalLayout_8">
                       <item>
                        <layout class="QVBoxLayout" name="verticalLayout_12">
                         <item>
                          <widget class="QLabel" name="label_4">
                           <property name="text">
                            <string>Latitude :</string>
                           </property>
                          </widget>
                         </item>
                         <item>
                          <widget class="QLabel" name="label_2">
                           <property name="text">
                            <string>Longtitude :</string>
                           </property>
                          </widget>
                         </item>
                        </layout>
                       </item>
                      </layout>
                     </item>
                     <item>
                      <layout class="QVBoxLayout" name="verticalLayout_13">
                       <item>
                        <widget class="QLabel" name="Lbl_LatitudeD">
                         <property name="text">
                          <string>0.0000</string>
                         </property>
                        </widget>
                       </item>
                       <item>
                        <widget class="QLabel" name="Lbl_LongtitudeD">
                         <property name="text">
                          <string>0.0000</string>
                         </property>
                        </widget>
                       </item>
                      </layout>
                     </item>
                     <item>
                      <layout class="QVBoxLayout" name="verticalLayout_14">
                       <item>
                        <widget class="QPushButton" name="Btn_SelectFromTableD">
                         <property name="text">
                          <string>Selected</string>
                         </property>
                        </widget>
                       </item>
                       <item>
                        <widget class="QPushButton" name="Btn_SelectMyLocationD">
                         <property name="text">
                          <string>Current</string>
                         </property>
                        </widget>
                       </item>
                      </layout>
                     </item>
                    </layout>
                   </item>
                   <item>
                    <layout class="QHBoxLayout" name="horizontalLayout_10">
                     <item>
                      <layout class="QVBoxLayout" name="verticalLayout_15">
                       <item>
                        <widget class="QLabel" name="label_6">
                         <property name="text">
                          <string>Distance :</string>
                         </property>
                        </widget>
                       </item>
                       <item>
                        <widget class="QLabel" name="label_5">
                         <property name="text">
                          <string>Degree :</string>
                         </property>
                        </widget>
                       </item>
                      </layout>
                     </item>
                     <item>
                      <layout class="QVBoxLayout" name="verticalLayout_16">
                       <item>
                        <widget class="QLabel" name="Lbl_Distance">
                         <property name="text">
                          <string>0.0000</string>
                         </property>
                        </widget>
                       </item>
                       <item>
                        <widget class="QLabel" name="Lbl_Degree">
                         <property name="text">
                          <string>0</string>
                         </property>
                        </widget>
                       </item>
                      </layout>
                     </item>
                    </layout>
                   </item>
                   <item>
                    <layout class="QHBoxLayout" name="horizontalLayout_9">
                     <item>
                      <widget class="QPushButton" name="Btn_DistanceCalculate">
                       <property name="text">
                        <string>Calculate</string>
                       </property>
                      </widget>
                     </item>
                     <item>
                      <widget class="QPushButton" name="Btn_DistanceCancel">
                       <property name="text">
                        <string>Cancel</string>
                       </property>
                      </widget>
                     </item>
                    </layout>
                   </item>
                  </layout>
                 </item>
                </layout>
               </widget>
              </item>
             </layout>
            </item>
           </layout>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>558</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
